.github/workflows/create-release.yml

```diff
@@ -59,27 +50,64 @@ jobs:
         echo "# Platform-specific torch requirements (See SCT Issue #2745)" >> requirements-freeze.txt
         grep "torch" requirements.txt >> requirements-freeze.txt
 
+    - name: Upload requirements-freeze.txt
+      uses: actions/upload-artifact@v3
+      with:
+        name: requirements-freeze
+        path: requirements-freeze.txt
+
+  test-release:
+    # The purpose of this step is to run tests using the requirements-freeze file
+    # In 99% of cases, this will be identical to running tests using `requirements.txt`. But, sometimes
+    # there are unique bugs that arise from using the `requirements-freeze.txt` file.
+    needs: generate-requirements-freeze
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15, windows-2019]
```

Comment:
Huzzah! The [`windows-2019`](https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/7286144257?check_suite_focus=true) workflow did in fact fail with the following exception:

```
Check if msvc-runtime is installed..................[FAIL]
An error occured while importing module msvc-runtime -> No module named 'msvc_runtime'
```

We can now rework the `generate-requirements-freeze` step to properly copy over the correct lines from `requirements.txt` to fix #3794.